tests/libtransmission/utils-test.cc

```diff
@@ -113,53 +113,53 @@ TEST_F(UtilsTest, trStrvStrip)
     EXPECT_EQ("test"sv, tr_strvStrip("test"sv));
 }
 
-TEST_F(UtilsTest, trStrvUtf8Clean)
+TEST_F(UtilsTest, strvReplaceInvalid)
 {
     auto in = "hello world"sv;
-    auto out = tr_strvUtf8Clean(in);
+    auto out = tr_strv_replace_invalid(in);
     EXPECT_EQ(in, out);
 
     in = "hello world"sv;
-    out = tr_strvUtf8Clean(in.substr(0, 5));
+    out = tr_strv_replace_invalid(in.substr(0, 5));
     EXPECT_EQ("hello"sv, out);
 
     // this version is not utf-8 (but cp866)
     in = "\x92\xE0\xE3\xA4\xAD\xAE \xA1\xEB\xE2\xEC \x81\xAE\xA3\xAE\xAC"sv;
-    out = tr_strvUtf8Clean(in);
+    out = tr_strv_replace_invalid(in, '?');
     EXPECT_TRUE(std::size(out) == 17 || std::size(out) == 33);
```

Comment:
I suspect there's no difference in behavior now (compared to old implementation that could have acted differently based on current locale), so one of the equality checks should be dropped.

Reply:
Fixed in a62195e